app/test_main.py

```diff
@@ -1,3 +1,80 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+class TestGetHumanAge:
```

Comment:
There is no need to organize these tests in class, you could write them as two separate functions and omit using `self` as a function argument.